source/MapPanel.h

```diff
@@ -144,7 +144,7 @@ class MapPanel : public Panel {
 
 	// Distance from the screen center to the nearest owned system,
 	// for use in determining which governments are in the legend.
-	std::map<const Government *, double> closeGovernments;
+	std::map<const Government *, unsigned int> closeGovernments;
```

Comment:
This comment and variable name should change, since they're no longer accurate.

Reply:
Done